components/card/card.css

```diff
@@ -0,0 +1,149 @@
+@import "../../assets/common-css/sidebar.css";
+@import "../../assets/common-css/navbar.css";
+@import "../../assets/common-css/containers.css";
+@import "../../document-css/doc.css";
+
+.card-head, 
+.card-type{
+    color: var(--font-color);
+    font-size: var(--head-type-font);
+}
+.card-head{
+    margin: -5px;
+}
+.card-type{
+    margin-top: var(--type-margin);
+}
+.card-content{
+    padding: var(--content-padding);
+    font-size: var(--content-font);
+    color: var(--primary-color);
+}
+.card-container{
+    display: flex;
+    flex-direction: row;
+    gap: var(--content-gap);
+    align-items: center;
+    justify-content: center;
+    padding: 1.875rem;
+    background-color: var(--eg-container-bg);
+    border-radius: var(--border-radius);
+    flex-wrap: wrap;
+    margin-top: var(--container-margin-top);
+
+}
+
+.complete-card{
+    display: flex;
+    /* flex-basis: 500px; */
+    max-width: 25rem;
+    /* height: 800px; */
+    flex-direction: column;
+    /* border: 1px solid var(--secondary-color); */
+    padding-bottom: 0.9375rem;
+    border-radius: var(--border-radius);
+    background-color: var(--background-color);
+}
+.card-image{
+    width: 100%;
+    border-top-left-radius: var(--border-radius);
+    border-top-right-radius: var(--border-radius);
+}
+.footer{
+    display: flex;
+    flex-direction: row;
+    margin: 5px 10px;
+}
+.icon-cont{
+    display: flex;
+    flex-direction: row;
+    margin-left: auto;
+    gap: 1.5625rem;
+    justify-content: space-around;
+}
+.main-info-cont,
+.sub-info-cont{
+    padding: 0px 10px 0px 10px;
+    color: var(--font-color);
+}
+h3{
+    color: var(--font-color);
+    margin-bottom: none;
+    margin: 10px 0px 0px 0px;
+    padding-bottom: none;
+}
+.sm-txt{
+    margin: 0px 0px 10px 0px; 
+    font-size: smaller;
+    padding-top: none;
+    color: var(--secondary-color);
+}
+.button, 
+.card-icon{
+    border: none;
+    border-radius: var(--border-radius);
+    font-family: "Montserrat", sans-serif;
+    font-size: 1rem;
+    color: var(--primary-color);
+}
+.button{
+    margin: 3px;
+    /* background-color: #acbbce; */
+    padding: 10px;
+    font-weight: 500;
+}
+.button:hover{
+    /* text-decoration: underline; */
+    color: var(--font-color);
+    background-color: #acbbce;
+}
+
+.relative{
+    position: relative;
+}
+.badge, 
+.dismiss-badge{
+    
+    position: absolute;
+    right: 0;
+    top:0;
+    padding: 2px 0.4rem 2px 0.4rem;
```

Comment:
Can use short-hand property like : padding: 2px 0.4rem;

Reply:
Okay, will use that from now on.